app/src/main/java/com/piwniczna/mojakancelaria/activities/MainActivity.kt

```diff
@@ -70,14 +80,28 @@ class MainActivity : AppCompatActivity(), NavigationView.OnNavigationItemSelecte
             R.id.nav_info -> showAboutDialog()
             R.id.nav_exit -> exit()
             R.id.nav_backup -> openFragment(BackupFragment(), stack=true)
-            //todo: pozostałe
+            R.id.nav_new_letter -> showLetterDialog()
+            R.id.nav_avizo -> openFragment(LettersFragment(false))
+            R.id.nav_sent -> openFragment(LettersFragment(true))
+
         }
         drawer.closeDrawer(GravityCompat.START)
 
 
         return true
     }
 
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        when(item.itemId) {
+            // Home button id - dunno why R.id.home wont work...
```

Comment:
put TODO here to investigate it maybe in future

Reply:
pffff

Reply:
Ok